_i18n/it/resources/moneropedia/account.md

```diff
@@ -44,39 +53,40 @@ List of available languages for your wallet's seed:
 8 : 日本語
 9 : 简体中文 (中国)
 10 : Esperanto
-Enter the number corresponding to the language of your choice: 1
-Generated new wallet: 4B15ZjveuttEaTmfZjLVioPVw7bfSmRLpSgB33CJbuC6BoGtZrug9TDAmhZEWD6XoFDGz55bgzisT9Dnv61sbsA6Sa47TYu
-view key: 4130fa26463d9451781771a8baa5d0b8085c47c4500cefe4746bab48f1d15903
+11 : Lojban
+Enter the number corresponding to the language of your choice: 4
+Generated new wallet: 48zfLeXXJqYVMdbRNbMQFLMDdgtkdQA3QPqaps4dbKGyVazV8C1H7AQifXbEyMeG49DzyMFphUM3XPavayao1AB2VEpBUkY
+View key: e7afaf6ad061d9b92b3901bcddd3e248973c129d40df5b055cfe05e7df961305
 **********************************************************************
-Your wallet has been generated.
-To start synchronizing with the daemon, use "refresh" command.
-Use "help" command to see the list of available commands.
-Always use "exit" command when closing monero-wallet-cli to save your
-current session's state. Otherwise, you might need to synchronize 
+Your wallet has been generated!
```

Comment:
Are this lines left untranslated on purpose?

Reply:
The interface has been left in english because at the time I didn't have the possibility to run a cli on the v0.12 in italian. By the way, are you sure the cli is totally available in italian?...

Reply:
The best (at least, what i did for French) is to open the translation file from the monero repository, and search & replace with it. This way you're sure, and you don't need a working node/wallet.

Reply:
Hi @el00ruobuob , thanks for the clue. Since the file belongs to Moneropedia and I'm translating it in background, I'll open a specific PR with the refresh of this file too.
Thanks again.